gcp/vars.tf

```diff
@@ -0,0 +1,50 @@
+variable "region" {
+  default = "us-west1"
+}
+
+variable "zone" {
+  description = "The zone where the cluster will be deployed [a,b,...]"
+  default     = "a"
+}
+
+variable "subnet" {
+  description = "The name of the existing subnet where the machines will be deployed"
+}
+
+variable "nodes" {
+  description = "The number of nodes to deploy."
+  type        = number
+  default     = "1"
+}
+
+variable "disks" {
+  description = "The number of local disks on each machine."
+  type        = number
+  default     = 1
+}
+
+variable "image" {
+  # See https://cloud.google.com/compute/docs/images#os-compute-support
+  # for an updated list.
+  default = "ubuntu-os-cloud/ubuntu-1804-lts"
+}
+
+variable machine_type {
```

Comment:
Like the aws module, should we separate out the machine type for the redpanda nodes vs the monitoring nodes? Also, I don't believe we need NVME drives for Prometheus/grafana.

Reply:
I agree, but the idea was to get this out ASAP so Saucelabs could use it